### ISSUE TITLE
Global search not returning any results

### ISSUE THREAD

**Issue by frederic-gendebien-es** [event-catalog/eventcatalog#574]
## Description

**GIVEN** I have quite a lot of events
**WHEN** I use the global search to find some events
**THEN** No results come back

## How to reproduce?

1. Create a new catalog

```shell
npx @eventcatalog/create-eventcatalog@latest catalog
cd catalog
npm run dev
```

2. Browse the catalog

[catalog](http://localhost:3000)

## Environment

- Browser: Arc

**Comment by boyney123** [comment]
OK thanks, can you try `npm run build` and then retry dev again? I think things are only indexed at build.... but I need to check.

https://github.com/event-catalog/eventcatalog/blob/907d4ef347ddcfe355778c211cd5cf1829736e35/astro.config.mjs#L55

**Comment by boyney123** [comment]
Ah think I found the issue, pagefind it looking for the output in a different place. Will try and fix this :)

**Comment by boyney123** [comment]
Should be fixed in `2.0.26`,
